stubs/python-periphery/periphery/mmio.pyi

```diff
@@ -0,0 +1,27 @@
+from ctypes import c_void_p
+from types import TracebackType
+
+long = int
+
+class MMIOError(IOError): ...
+
+class MMIO:
+    def __init__(self, physaddr: int | long, size: int | long, path: str = ...) -> None: ...
```

Comment:
Just change `int | long` to `int` (here and below) as it's basically just an alias for `int | int`.